### DISCUSSION TITLE
Issue sharing github repo link

### DISCUSSION THREAD

**Discussion by harrisonsmith32** [community/community#35191] [New to GitHub]
Hi, I am new to github and attempting to share my repository to ask a question about a project that I am working on. The link to the repo is working within my browser, but the person trying to access it is seeing a 404 page. Also, when I click on the embedded link within the posted discussion, it displays 404 for me as well. The repository is listed as public. Any help would be appreciated!

**Comment by airtower-luna** [comment]
That sounds like the link got broken wherever you posted it, but of course I can't confirm that without seeing the actual link. You might want to check if the link you wanted to post and the one in the discussion (right click, "copy link" in most browsers) are actually exactly the same.

**Reply by harrisonsmith32** [reply]
Hi, thanks for the help. I have verified that both of the links are exactly the same, but the issue is still persisting.  @airtower-luna

**Comment by Sleepy4k** [comment]
> Hi, I am new to github and attempting to share my repository to ask a question about a project that I am working on. The link to the repo is working within my browser, but the person trying to access it is seeing a 404 page. Also, when I click on the embedded link within the posted discussion, it displays 404 for me as well. The repository is listed as public. Any help would be appreciated!

From what I read in this case, it looks like you created a private repository so that people who try to access the repository link receive 404 pages

**Reply by harrisonsmith32** [reply]
Hi, thanks for reaching out. I have verified that the repository is listed as public in the repo settings, but the issue is still persisting.  @Sleepy4k

**Reply by Sleepy4k** [reply]
> Hi, thanks for reaching out. I have verified that the repository is listed as public in the repo settings, but the issue is still persisting. @Sleepy4k

ah okey, if you allow it can i see your "repository"?. Because it seems abnormal

**Reply by harrisonsmith32** [reply]
yes, it is listed as hs-fcc-blocklchain-learning under my profile. Here is the link that I have attempted to share: [https://github.com/harrisonsmith32/hs-fcc-blocklchain-learning.git](url)

**Reply by Sleepy4k** [reply]
> yes, it is listed as hs-fcc-blocklchain-learning under my profile. Here is the link that I have attempted to share: [https://github.com/harrisonsmith32/hs-fcc-blocklchain-learning.git](url)

it's good for me working as well, maybe share this link in your messager platform for share it to your team or friend just copy paste it. But yea its have problem when click it in this reply they redirect us to https://github.com/orgs/community/discussions/url and i try to copy paste it in browser and good
https://github.com/harrisonsmith32/hs-fcc-blocklchain-learning

**Reply by harrisonsmith32** [reply]
Problem resolved, thank you!

**Reply by Sleepy4k** [reply]
> Problem resolved, thank you!

your welcome, if your feel this problem solved you can mark this section as solved section thanks
